vip-helpers/vip-migrations.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+add_action( 'vip_post_migration', 'vip_post_migration' );
+
+function vip_post_migration() {
+	if ( is_multisite() ) {
+		$sites = get_sites();
+
+		foreach( $sites as $site ) {
+			switch_to_blog( $site['blog_id'] );
+
+			vip_run_post_migration_cleanup();
+
+			restore_current_blog();
+		}
+
+		return true;
+	} else {
+		return vip_run_post_migration_cleanup();
+	}
+}
+
+function vip_run_post_migration_cleanup() {
+	/**
+	 * Fires on migration cleanup
+	 *
+	 * Migration cleanup runs on VIP Go during the initial site setup
+	 * and after database imports. This hook can be used to add additional
+	 * cleanup for a given site.
+	 */
+	do_action( 'vip_go_migration_cleanup' );
+
+	vip_delete_db_transients();
+
+	wp_cache_flush();
+
+	vip_connect_jetpack();
+	vip_connect_vaultpress();
+
+	return true;
+}
+
+function vip_delete_db_transients() {
+	global $wpdb;
+
+	return $wpdb->query(
+		"DELETE FROM $wpdb->options
+		WHERE option_name LIKE '\_transient\_%'
+		OR option_name LIKE '\_site\_transient\_%'"
+	);
+}
+
+function vip_connect_jetpack() {
+	return \WP_CLI::runcommand( sprintf( 'jetpack-start connect --url=%s', home_url() ) );
```

Comment:
Is this going to work in a non-WP_CLI context?

Reply:
🤷‍♂️

Reply:
I think this will work:

```
$ wp shell
wp> \WP_CLI::runcommand( sprintf( 'jetpack-start connect --url=%s', home_url() ) );
- Jetpack is already connected
-- Everything looks good!
Success: All done! Welcome to Jetpack! ✈️️✈️️✈️️
=> phar:///root/roles/docker-container-vip-web/usr/local/bin/wp/vendor/wp-cli/shell-command/src/WP_CLI/REPL.php:43:
NULL
```

Reply:
`wp shell` is a WP_CLI context though.

Reply:
I think we should at least wrap these in `function_exists()` checks.

Reply:
Added a check in f16530a